code/go/0chain.net/smartcontract/storagesc/challenge_integration_tests.go

```diff
@@ -0,0 +1,128 @@
+//go:build integration_tests
```

Comment:
it's not correct to overwrite here, we need to operate with finalized state, but not speculative